Userland/Libraries/LibJS/Bytecode/Op.h

```diff
@@ -162,6 +162,24 @@ class NewObject final : public Instruction {
     void replace_references_impl(BasicBlock const&, BasicBlock const&) { }
 };
 
+class NewRegExpLiteral final : public Instruction {
```

Comment:
`Literal` feels like a parsing-stage concept, I think we can just call this `NewRegExp` :)